epytope/EpitopePrediction/External.py

```diff
@@ -6998,7 +8327,9 @@ def parse_external_result(self, file):
                 if not row[0].isdigit():
                     continue
             
-                epitope, allele, comb_score = Peptide(row[PeptideIndex.NETCTLPAN_1_1]), Allele(row[HLAIndex.NETCTLPAN_1_1]), float(row[ScoreIndex.NETCTLPAN_1_1])
+                epitope = row[PeptideIndex.NETCTLPAN_1_1]
+                allele = row[HLAIndex.NETCTLPAN_1_1].replace('*','')
```

Comment:
A comment here would help to remember why it is necessary to change the string here.